troposphere/static/js/components/common/InteractiveDateField.react.js

```diff
@@ -47,13 +47,38 @@ define(function (require) {
     },
 
     render: function () {
-      var labelEl = this.props.labelText ? (<label>{this.props.labelText}</label>) : "";
+      var labelEl = this.props.labelText ? (<label htmlFor="interactive-date-field">{this.props.labelText}</label>) : null;
+      var btnStyle = {
+        lineHeight: '1.31' // I know this is quite heinous ...
+      };
+      var lastBtnStyle = {
+        borderTopRightRadius: '5px',
+        borderBottomRightRadius: '5px'
+      };
+      _.extend(lastBtnStyle, btnStyle);
+
```

Comment:
Good use of inline styles here. Others should note that this is how we should deal with situations like this that require hacking bootstrap as it doesn't pollute the cascade and mess with other code that might be using these classes.